shared/src/main/java/uk/gov/di/authentication/shared/entity/ErrorResponse.java

```diff
@@ -59,7 +59,14 @@ public enum ErrorResponse {
             "System is blocked from sending any email verification codes for changing how to receive security codes"),
     ERROR_1048(
             1048,
-            "User entered invalid email verification code for changing how to receive security codes too many times");
+            "User entered invalid email verification code for changing how to receive security codes too many times"),
+    ERROR_1049(1049, "Redirect URI is empty"),
```

Comment:
This sort of goes with my other comment about using the serialisation service. If you did, you might lose the granularity of all of these error codes...but not sure that matters (I personally don't think so, but putting it out there as a comment for you and others to weigh in on)

Reply:
Agreed, since using the serialisation service class which provides out of the box validation, it sought of makes this redundant